src/docs/content/docs/usage/multi_dc_distributed.md

```diff
@@ -0,0 +1,76 @@
+
+The `datacenterAvailability` setting in the Reaper YAML file indicates to Reaper its deployment in relation to cluster data center network locality.
+
+## Multiple Reaper instances with JMX accessible for all DCs
+
+In the case where the JMX port is accessible (with or without authentication) from the running Reaper instance for all nodes in all DCs, it is possible to have multiple instances of Reaper handle one or multiple clusters by using the following setting in the configuration yaml file :  
+
+```
+datacenterAvailability: ALL
+```
+
+
+{{< screenshot src="/img/singlereaper-multidc-all.png">}}
+
+{{< /screenshot >}}
+
+Reaper must be able to access the JMX port (7199 by default) and port 9042 if the cluster is also used as Cassandra backend, on the local DC.
+
+The keyspaces must be replicated using NetworkTopologyStrategy (NTS) and have replicas at least on the DC Reaper can access through JMX. Repairing the remote DC will be handled internally by Cassandra.
+
+## Multiple Reaper instances with JMX accessible for limited DCs
+
+In the case where the JMX port is accessible (with or without authentication) from the running Reaper instance for all nodes in only some of the DCs, it is possible to have multiple instances of Reaper handle one or multiple clusters by using the following setting in the configuration yaml file :  
+
+```
+datacenterAvailability: LOCAL
+```
+
+Be aware that this setup will not allow to handle backpressure for those remote DCs as JMX metrics (pending compactions, running repairs) from those remote nodes are not made available to Reaper.
+
+If multiple clusters are registered in Reaper it is required that the Reaper instances can access all nodes in at least one datacenter in each of the registered clusters.
+
+If all Reaper instances that have JMX access to a specific datacenter stop then backpressure for the nodes in that datacenter is not handled. 
+
+`LOCAL` mode allows you to register multiple clusters in a distributed Reaper installation. `LOCAL` mode also allows you to prioritise repairs running according to their schedules over worrying about the load on remote and unaccessible datacenters and nodes. 
+
+
+{{< screenshot src="/img/singlereaper-multidc-local.png">}}
+
+{{< /screenshot >}}
+
+Reaper must be able to access the JMX port (7199 by default) and port 9042 if the cluster is also used as Cassandra backend, on the local DC.
+
+The keyspaces must be replicated using NetworkTopologyStrategy (NTS) and have replicas at least on the DC Reaper can access through JMX. Repairing the remote DC will be handled internally by Cassandra.
```

Comment:
> Repairing the remote DC will be handled internally by Cassandra.

Only if some nodes dont have a Reaper instance that can access them.

Reply:
i just took the sentence out, it was a bit confusing. lmdwyt.